### PR TITLE
Game test

### PR DESCRIPTION
**Pull request by skoppers**

### CODE REVIEW

File: src/main/resources/Options/OptionsPlayer1.xml [hunk 1]
```diff
@@ -1,9 +1 @@
...
-<options>
-	<volume>100</volume>
-	<controls>
-		<left>Left</left>
-		<right>Right</right>
-		<shoot>Up</shoot>
-	</controls>
-</options>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><options><volume>0.0</volume><controls><left>Left</left><right>Right</right><shoot>Up</shoot></controls></options>
```
Comment (cbilstra) [inline src/main/resources/Options/OptionsPlayer1.xml]: Does this line layout possibly give problems?

Comment (skoppers) [inline src/main/resources/Options/OptionsPlayer1.xml]: It's the layout from the the writer, I commited this by accident. The content is the same, only the layout is different.

Comment (cbilstra) [inline src/main/resources/Options/OptionsPlayer1.xml]: but is it problematic?

Comment (skoppers) [inline src/main/resources/Options/OptionsPlayer1.xml]: no

**Pull request merged**